docs/resources/client/models/modelsystem.md

```diff
@@ -66,6 +66,8 @@ Minecraft's render engine recognizes a total of 8 perspective types (9 if you in
 | `FIXED`                   | `"fixed"`                 | Item frames                                                                                                      |
 | `NONE`                    | `"none"`                  | Fallback purposes in code, should not be used in JSON                                                            |
 
+NeoForge allows the `ItemDisplayContext` to be [extended] for use in custom render calls. Modded `ItemDisplayContext` may specify a fallback transform to use if none is specified in the model. Otherwise, behavior will be the same as Vanilla.
```

Comment:
```suggestion
NeoForge allows the `ItemDisplayContext` to be [extended] for use in custom render calls. Modded `ItemDisplayContext`s may specify a fallback transform to use if none is specified in the model. Otherwise, behavior will be the same as vanilla.
```
We don't usually capitalize vanilla elsewhere in the docs.